examples/java/src/test/java/org/apache/beam/examples/WindowedWordCountIT.java

```diff
@@ -67,6 +68,14 @@ private void testWindowedWordCountPipeline(boolean isStreaming) throws IOExcepti
         TestPipeline.testingPipelineOptions().as(WindowedWordCountITOptions.class);
     options.setStreaming(isStreaming);
 
+    // Note: currently unused because the example writes to BigQuery, but WindowedWordCount.Options
+    // are tightly coupled to WordCount.Options, where the option is required.
+    options.setOutput(IOChannelUtils.resolve(
```

Comment:
Perhaps not at the right level of abstraction. I'd prefer if test infrastructure (TestPipeline, TestPipelineOptions, TestRunner, etc.) could hold this functionality, and the test calls it to get output location.